src/foam/nanos/script/TestRunnerScript.js

```diff
@@ -76,8 +77,13 @@ foam.CLASS({
         loggerFilter.setLogInfo(false);
         loggerFilter.setLogWarning(false);
 
+        TestConfig testConfig = (TestConfig) x.get("testConfig");
```

Comment:
@jlhughes TestConfig is used here.

Reply:
I could rename it TestRunnerConfig instead of just TestConfig

Reply:
yes please.

Reply:
rename affected ..